config/config.go

```diff
@@ -355,11 +323,47 @@ func loadConfigFile(configFile string, config *MenderShellConfig, filesLoadedCou
 		return err
 	}
 
+	if err := checkforDeprecatedFields(configFile); err != nil {
+		log.Errorf("Error loading configuration from file: %s (%s)", configFile, err.Error())
+		return err
+	}
+
 	*filesLoadedCount++
 	log.Info("Loaded configuration file: ", configFile)
 	return nil
 }
 
+func checkforDeprecatedFields(configFile string) error {
```

Comment:
this one does not have units -- hence the coverage decrease I guess.

Reply:
Most of it [is tested](https://github.com/mendersoftware/mender-connect/pull/46/files#diff-373f5578b8f362a364b50b0d01919a57da64c128547ca10d49df8ae422237c82R372), and so it is reflected in the [coverage report](https://coveralls.io/jobs/89987608/source_files/14424565085#L336).